src/packer.pkr.hcl

```diff
@@ -151,11 +151,162 @@ build {
   provisioner "powershell" {
     # Wait 90 seconds before executing the check-defender.ps1 powershell script.
     # This gives a generous grace period between restarting Windows and running the second provisioner.
-    pause_before = "90s"
-    scripts      = ["src/powershell/check-defender.ps1"]
+    pause_before = "30s"
+    scripts = [
+      "src/powershell/check-defender.ps1",
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/install-chocolatey.ps1"
+    ]
   }
 
   provisioner "powershell" {
-    scripts = ["src/powershell/enable-rdp.ps1"]
+    inline = ["mkdir C:\\packages"]
+  }
+
+  provisioner "file" {
+    source      = "src/packages/"
+    destination = "C:\\packages"
+  }
```

Comment:
Even though these are simple provisioners, you still should add a brief comment explaining each.

Reply:
Addressed here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/46f59aa6e9b802041bd7fc37fbce1cb9f9aa0482